libraries/SITL/SIM_JSBSim.cpp

```diff
@@ -198,9 +198,11 @@ bool JSBSim::start_JSBSim(void)
         }
         char *logdirective;
         char *script;
+        char *nice;
 
         asprintf(&logdirective, "--logdirectivefile=%s", jsbsim_fgout);
         asprintf(&script, "--script=%s", jsbsim_script);
+        asprintf(&nice, "--nice=%.8f", 10*1e-9);
```

Comment:
Set as 10ns since it gave good result on my system, can be increased or decreased after testing on other systems